admin/config-ui/class-configuration-page.php

```diff
@@ -214,16 +215,8 @@ private function remove_notification() {
 	 * @return Yoast_Notification
 	 */
 	private static function get_notification() {
-		$message  = __( 'The configuration wizard helps you to easily configure your site to have the optimal SEO settings.', 'wordpress-seo' );
-		$message .= '<br/>';
-		$message .= sprintf(
-			/* translators: %1$s resolves to Yoast SEO, %2$s resolves to the starting tag of the link to the wizard, %3$s resolves to the closing link tag */
-			__( 'We have detected that you have not finished this wizard yet, so we recommend you to %2$sstart the configuration wizard to configure %1$s%3$s.', 'wordpress-seo' ),
-			'Yoast SEO',
-			'<a href="' . admin_url( '?page=' . self::PAGE_IDENTIFIER ) . '">',
-			'</a>'
-		);
-
+		$note = new Wizard_Notification();
+		$message = $note->get_notification_message( 'continue' );
 		$notification = new Yoast_Notification(
```

Comment:
Although it appears we don't have a PHPCS rule to enforce it, I think we have some coding style best practice to align this kind of things for better readability.